Gruntfile.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Generated on 2013-12-04 using generator-backbone-amd 0.0.4
+//
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const extract = require('jsxgettext-recursive');
+
+// # Globbing
+// for performance reasons we're only matching one level down:
+// 'test/spec/{,*/}*.js'
+// use this if you want to recursively match all subfolders:
+// 'test/spec/**/*.js'
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.loadNpmTasks('grunt-contrib-copy');
+
+  grunt.initConfig({
+    copy: {
+      strings: {
+        files: [{
+          expand: true,
+          flatten: true,
+          cwd: path.join(__dirname, 'node_modules', 'fxa-content-server-l10n', 'locale', 'templates', 'LC_MESSAGES'),
+          dest: path.join(__dirname),
```

Comment:
Not sure if we need the `path.join()` dance if there is only one directory (same w/ line 45 below)